examples/pytorch_lightning_distributed/swav.py

```diff
@@ -49,21 +47,16 @@ def configure_optimizers(self):
 
 model = SwaV()
 
-# we ignore object detection annotations by setting target_transform to return 0
-pascal_voc = torchvision.datasets.VOCDetection(
-    "datasets/pascal_voc", download=True, target_transform=lambda t: 0
-)
 transform = SwaVTransform()
-dataset = LightlyDataset.from_torch_dataset(pascal_voc, transform=transform)
+dataset = pascal_voc = torchvision.datasets.VOCDetection(
+    "datasets/pascal_voc", download=True, transform=transform
+)
```

Comment:
```suggestion
dataset = torchvision.datasets.VOCDetection(
    "datasets/pascal_voc", download=True, transform=transform
)
```

Is `target_transform` not needed anymore?